solidity/random-beacon/test/Utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ethers } from "hardhat"
+import { expect } from "chai"
+import { hashDKGMembers } from "./utils/dkg"
+
+describe("utils/dkg", () => {
+  context("hashDKGMembers", () => {
+    const members = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
Could we make it more difficult by changing the values not to be `0 ... 9`? We expect indices in `misbehavedMembers` array, so I don't want to confuse them.

```suggestion
    const members = [100, 101, 102, 103, 104, 105, 106, 107, 108, 109]
```